test/interpreter/criterion-computations.spec.ts

```diff
@@ -305,8 +305,8 @@ describe( 'big test', () => {
     expect(engine.getCellValue(adr('A39'))).toEqual(1)
     // expect(engine.getCellValue(adr('A40'))).toEqual(0)
     // expect(engine.getCellValue(adr('A41'))).toEqual(1)
-    expect(engine.getCellValue(adr('A42'))).toEqual(3)
-    expect(engine.getCellValue(adr('A43'))).toEqual(3)
+    expect(engine.getCellValue(adr('A42'))).toEqual(1)
+    expect(engine.getCellValue(adr('A43'))).toEqual(1)
```

Comment:
Can you explain why those results have changed?

Reply:
Semantic of how we interpret `'` changed.

Reply:
So we just changed the test that were previously compatible with Excel and Google Sheets without adding a comment about incompatibility with Product 1 and Product 2?